src/lib/types.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type UnknownPropertiesObjType = { [key: string]: any; };
```

Comment:
@juanfabrega Is this where we want to keep the truly global types?

Reply:
@JCarmine yeah this should work.